lib/scout_apm/plugs/controller_timer.ex

```diff
@@ -1,6 +1,6 @@
 defmodule ScoutApm.Plugs.ControllerTimer do
   alias ScoutApm.Internal.Layer
-  alias ScoutApm.{Context, TrackedRequest}
+  alias ScoutApm.{Context, TrackedRequest, Config}
```

Comment:
I think we should swap the boolean to be `:include_application_name` (and defaulting to false) and move the config from global config to the plug options. I could see use cases where one may want to enable or disable on a per-controller basis. Something like:

```elixir
def call(conn, opts) do
  # ...
    TrackedRequest.start_layer("Controller", action_name(conn, opts))
  # ...
end

def action_name(conn, opts) do
  # ...
  include_app_name = Keyword.get(opts, :include_application_name, false)
  # ...
  |> trim_app_module_name(include_app_name)
  # ...
end

defp trim_app_module_name(parts, include_app_name) do
  if include_app_name do
    parts
  else
    Enum.drop(parts, 1)
  end
end
```

Since most users are adding the plug via `use ScoutApm.Instrumentation`, we should allow the configuration to be pulled in from there as well. Ex:

```elixir
defmodule ScoutApm.Instrumentation do
  defmacro __using__(opts) do
    timer_options = Keyword.take(opts, [:include_application_name])

    quote do
      plug(ScoutApm.DevTrace.Plug)
      plug(ScoutApm.Plugs.ControllerTimer, unquote(timer_options))
    end
  end
end
```